### ISSUE TITLE
Use a custom wifi handler

### ISSUE THREAD

**Issue by VedantParanjape** [VedantParanjape/esp-wifi-logger#12]
Using protocol-common example for connecting to wifi is forbidden. It is possible to implement a simple wifi handler for the same. It will also remove dependency of protcol_examples_common example.

CC: @shivam5594

**Comment by shivam5594** [comment]
Yes, true. I have already implemented that locally. Let me see if I can embed that

**Comment by VedantParanjape** [comment]
Cool, I can use that. I need it for one more project I am doing

**Comment by VedantParanjape** [comment]
Hi, any progress ?

**Comment by shivam5594** [comment]
Haven't worked on it.

**Comment by VedantParanjape** [comment]
Ohk

**Comment by VedantParanjape** [comment]
Hi, you seem to be busy. should I close this issue?

**Comment by shivam5594** [comment]
Yes I am little busy right now. You can either move it to you instead of closing. Someone else might do it.

**Comment by VedantParanjape** [comment]
@shivam5594 https://github.com/Open-Authenticator/wifi-handler-component I wrote this finally, I need to document it still. But all functions have doc strings.

**Comment by shivam5594** [comment]
That's great. I'll check it soon. 👏
